builtin/credential/cert/path_certs.go

```diff
@@ -156,6 +165,47 @@ func (b *backend) pathCertWrite(
 	allowedNames := d.Get("allowed_names").([]string)
 	requiredExtensions := d.Get("required_extensions").([]string)
 
+	var resp logical.Response
+
+	// Parse the ttl (or lease duration)
+	systemDefaultTTL := b.System().DefaultLeaseTTL()
+	ttl := time.Duration(d.Get("ttl").(int)) * time.Second
+	if ttl == 0 {
+		ttl = time.Second * time.Duration(d.Get("lease").(int))
```

Comment:
minor, the previous statement is ttl * time.Second while this one is time.Second * lease.

Reply:
It should definitely be `ttl`, nice catch!

Reply:
That is for backwards compatibility with the old term for it, lease.  I was just commenting on the ordering of the time math which makes it harder to scan through.

Reply:
Yea you're right. I'll swap the ordering and keep lease.